examples/scalajs-react/src/main/scala/todomvc/CTodoList.scala

```diff
@@ -70,62 +83,64 @@ object CTodoList {
           <.input(
             ^.className     := "new-todo",
             ^.placeholder   := "What needs to be done?",
-            ^.onKeyDown  ~~>? handleNewTodoKeyDown _,
+            ^.onKeyDown   ==>? cbs.value().handleNewTodoKeyDown,
```

Comment:
Just wondering, is it necessary/correct/good style to call `value()` as opposed to treating it as a value -> `value`?